src/generate.rs

```diff
@@ -14,11 +15,54 @@ pub enum GenMode {
     GetMut,
 }
 
-fn attr_name(attr: &Attribute) -> Option<Ident> {
-    attr.interpret_meta().map(|v| v.name())
+pub fn attr_tuple(attr: &Attribute) -> Option<(Ident, Meta)> {
+    let meta = attr.interpret_meta();
+    meta.map(|v| (v.name(), v))
 }
 
-pub fn implement(field: &Field, mode: GenMode, params: GenParams) -> Tokens {
+pub fn parse_visibility(attr: Option<&Meta>, meta_name: &str) -> Option<Ident> {
+    match attr {
+        // `#[get = "pub"]` or `#[set = "pub"]`
+        Some(Meta::NameValue(MetaNameValue {
+            lit: Lit::Str(ref s),
+            ident,
+            ..
+        })) => {
+            if ident == meta_name {
+                let visibility = Ident::new(&s.value(), Span::call_site());
+                Some(visibility)
+            } else {
+                None
+            }
+        }
+        // `#[get(vis = "pub")]` or `#[set(vis = "pub")]`
```

Comment:
This seems unrelated, maybe you meant this bit for another PR?

Reply:
Yeah, I just forgot to remove it